src/clr-angular/data/tree-view/tree-node.spec.ts

```diff
@@ -186,6 +209,20 @@ export default function(): void {
         this.clarityDirective.expanded = false;
         expect(this.hostComponent.expanded).toBeFalse();
       });
+
+      it(
+        'offers a [(clrDisabled)] two-way binding',
+        fakeAsync(function(this: Context) {
+          this.hostComponent.disabled = true;
+          this.detectChanges();
+          expect(this.clarityDirective.disabled).toBe(true);
+          this.clarityDirective.disabled = false;
+          // We need fakeAsync and tick because the EventEmitter is asynchronous
+          tick();
+          // I don't know why Typescript forces me to cast this
+          expect<boolean>(this.hostComponent.disabled).toBe(false);
```

Comment:
Do we really need to cast here? I think a recent TS update fixed this inference issue `expect(this.hostComponent.disabled).toBe(false);`